packages/compass-saved-aggregations-queries/src/stores/open-item.ts

```diff
@@ -237,11 +237,19 @@ const openItem =
       }
     );
 
-    appRegistry.emit('open-namespace-in-new-tab', {
+    const emitData = {
       ...metadata,
-      aggregation: item.type === 'aggregation' ? item.aggregation : null,
-      query: item.type === 'query' ? item.query : null,
-    });
+      [item.type]: (item.type === 'aggregation')
+        ? {
+          ...item.aggregation,
+          namespace: metadata.namespace,
+        } : {
+          ...item.query,
+          _ns: metadata.namespace,
```

Comment:
Where is this `_ns` used? It looks like the main consumer of this `open-namespace-in-new-tab` looks explicitly for `namespace` and disregards `_ns`, looking at https://github.com/mongodb-js/compass/blob/29964950b387b3b68d605a798040bba90a219c24/packages/compass-collection/src/stores/index.ts#L170
Do the `aggregation` and `query` items open their contexts differently? I'm wondering if we should also be putting `namespace: metadata.namespace` here.

Reply:
Yeah totally!